ops/pebble.py

```diff
@@ -1083,6 +1083,7 @@ def __init__(self, socket_path=None, opener=None, base_url='http://localhost', t
         self.opener = opener
         self.base_url = base_url
         self.timeout = timeout
+        self.block_size = 102400  # hard-coded for now
```

Comment:
What about writing this as `100*1024` for clarity?

Reply:
Or replacing w/ the 8192 value used elsewhere...  I'll look into this.

Reply:
Looks like I already addressed this in a later change.